plugins/test/meson.build

```diff
@@ -46,3 +50,26 @@ shared_module('fu_plugin_invalid',
     plugin_deps,
   ],
 )
+
+shared_module('fu_plugin_test_ble',
```

Comment:
should this have `if get_option('bluez')` around it since we don't actually have another bluetooth implementation at the time?

Reply:
Not sure. At the moment the ble_test isn't actually a test, it's more to play with how we'd design a real plugin in the future.

Reply:
Thinking slowly; it's pointless to build this if we can't use it. fixed.